varats/paper/case_study.py

```diff
@@ -316,9 +320,9 @@ def halfnormal(num_samples):
         raise Exception('Unsupported SamplingMethod')
 
 
-@check_required_args(['extra_revs'])
+@check_required_args(['extra_revs', 'git_path', 'revs_per_year'])
 def generate_case_study(sampling_method: SamplingMethod, num_samples: int,
-                        cmap, project_name: str, case_study_version: int,
+                        cmap, proj_name: str, case_study_version: int,
```

Comment:
```suggestion
                        cmap, project_name: str, case_study_version: int,
```
I alwaysed tried to used `project_name` for variables and kwargs